src/core/Folder.java

```diff
@@ -0,0 +1,45 @@
+package core;
+
+import java.util.List;
+import java.io.File;
+import java.util.ArrayList;
+
+/**
+ * Created by Marcelo Loayza on 17/05/2016.
+ */
+public class Folder extends Item {
+
+    public Folder (File fileSystemItem) {
+        super(fileSystemItem);
+    }
+
+    /**
+     * Returns a list of files and folders that are located on an absolute path.
+     *
+     * @param   workingPath an absolute path giving the base location of the files and folders.
+     * @return                the list of files an folders.
```

Comment:
the list of files and folders is a correct description, but these files are strings or objects of the File type